Things.md

```diff
@@ -0,0 +1,139 @@
+This is a general guide about Things for all FullScreenShenanigans projects. This guide will show you how in-game Things are manipulated. Everything you see in the game (trees, blocks, the player, etc.) is a Thing. The Thing class is subclassed by a new class for everything (Tree class, Block class, Player class, etc.). These Things can be manipulated to change their properties, placements, etc.    
+  
+### Table of Contents
+1. [Manipulating Things](#manipulating-things)
+    1. [Shifting Things](#shifting-things)
+        1. [Shift](#shift)
+        2. [Set](#set)
+        3. [Midpoints](#midpoints)
+        4. [Update](#update)
+        5. [Slide](#slide)
+    2. [Opacity](#opacity)
+2. [GroupHoldr](#groupholdr)
+    1. [Getting Groups](#getting-groups)
+
+# Manipulating Things
+
+## Shifting Things  
+A Thing's placement on the screen is based on its `top`, `right`, `bottom`, and `left` properties. These define the bounding box for a Thing. A Thing is basically just a rectangle, with its height, width, and location defined by these properties. The coordinate system in which these Things are placed has an origin at the top left corner (0,0). Things are also scaled from these raw measurements to actual in-game measurements using a predefined `unitsize`. There are a number of ways to move Things, all of which are done using EightBitter and by manipulating these 4 properties.  
+
+#### Shift 
+The first is to use the shift functions; `shiftVert` and `shiftHoriz`. These move the Thing up and down or left and right.
```

Comment:
Since this is the first one, it would be good to provide a sample of doing this to the player.